content/ADR-213-quests-ui.md

```diff
@@ -0,0 +1,55 @@
+
+## Context, Reach & Prioritization
+
+The Quests System's user interface is a crucial component of the metaverse experience, as it enables users to interact with quests and track their progress. As part of the UI design process, we need to determine if we want to use the SDK to create the UI components and in that case, how. The Explorer is the term used for our frontend implementation, which can take different forms. We need to decide how to balance the use of the SDK with the needs of the Explorer to create a user-friendly and consistent UI across all implementations.
+
+It is important to prioritize the decision-making process to ensure that we have enough time to design and implement the UI components before the launch of the Quests System. While using the SDK to create the UI components has its advantages, there are also limitations to what we can achieve with it. For instance, while we can use the SDK to implement components that support our use case, there may be gaps that we cannot address. Ultimately, we need to find a solution that allows us to create a visually appealing and functional UI that meets the needs of our users.
+
+## Solution Space Exploration
+
+There are three potential solutions for designing the Quests System UI:
+
+Using pure SDK components: In this approach, we would create the UI using only the components provided by the SDK. While this approach provides consistency across implementations, there may be some gaps that we would need to fill by implementing additional components.
+
+Using SDK to communicate the UI state to the Explorer: In this approach, we would use the SDK to store the UI state and communicate it to the Explorer. This would reduce the amount of communication required between the Explorer and the backend to retrieve information needed to render the UI. However, we would need to ensure that the SDK's capabilities are sufficient for our use case.
+
+Using the Explorer for everything: In this approach, each Explorer implementation would be responsible for implementing the entire UI. While this approach provides the most flexibility, it also means that each implementation would need to re-do everything, which could result in inconsistent UIs across different implementations.
+
+Each of these approaches has its benefits and drawbacks, and we need to carefully consider them before making a decision.
+
+## Specification
+
+The technical specification for this approach will involve defining the specific data storage mechanisms that will be used in the SDK, as well as the frontend rendering logic. It will also require integration between the two systems to ensure that they are communicating effectively. The UI design will need to be developed in parallel with the technical implementation.
```

Comment:
```suggestion
The technical specification for this approach will involve defining the specific data storage mechanisms that will be used in the SDK, as well as the frontend rendering logic. It will also require integration between the two systems to ensure they are communicating effectively. The UI design SHOULD be developed in parallel with the technical implementation.
```